packages/api/src/modules/transaction/services.ts

```diff
@@ -640,4 +640,17 @@ export class TransactionService implements ITransactionService {
 
     return validStatus;
   }
+
+  listAll() {
+    return Transaction.createQueryBuilder('t')
+      .select([
+        't.id',
+        't.hash',
+        't.predicateId',
+        't.network',
+        't.createdBy',
```

Comment:
createdBy selects all infos on `UserModel`. 
Format the query better, to select only the id of this user